cmd/entrypoint/interesting_types.go

```diff
@@ -80,9 +80,9 @@ func GetInterestingTypes(ctx context.Context, serverTypeList []kates.APIResource
 		//
 		// Note that we pull `secrets.v1.` in to "K8sSecrets".  ReconcileSecrets will pull
 		// over the ones we need into "Secrets" and "Endpoints" respectively.
-		"Services":   {{typename: "services.v1."}},                             // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
-		"Endpoints":  {{typename: "endpoints.v1.", fieldselector: endpointFs}}, // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
-		"K8sSecrets": {{typename: "secrets.v1."}},                              // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
+		"Services":   {{typename: "services.v1."}},                                         // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
+		"Endpoints":  {{typename: "endpoints.v1.", fieldselector: endpointFs}},             // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
+		"K8sSecrets": {{typename: "secrets.v1.", fieldselector: "type=kubernetes.io/tls"}}, // New in Kubernetes 0.16.0 (2015-04-28) (v1beta{1..3} before that)
```

Comment:
Not all folks put tls secrets as `kubernetes.io/tls`. you could create a normal secret and use the `tls.crt` and `tls.key` fields. So this could end up being a breaking change.

Reply:
fair enough, how about the the 2nd proposal then to add the ability to configure a filter? It doesnt make sense to load and watch 10.000 resources in the case only a handful are required.
Where would that config go? Module crd?

Reply:
or actually from an env only as the module res would need to be loaded first otherwise (if thats not the case, didnt check the code yet)